docs/api/objects/post.md

```diff
@@ -25,26 +29,74 @@
 
 ```json
 {
-	"_id": "5c93e560-fddd-4146-98bc-edeb263d648f",
-	"post_id": "5c93e560-fddd-4146-98bc-edeb263d648f",
+	"_id": "8cacf340-182c-4cbe-98e4-807781ca5e22",
+	"attachments": [],
+	"author": {
+		"_id": "ShowierData9978",
+		"avatar": "gl4zm8aGO2U3MOUANDNMpnSA",
+		"avatar_color": "007ffd",
+		"flags": 0,
+		"pfp_data": 21,
+		"uuid": "4f4d986b-63ef-4f7d-9b8b-87368c6e0280"
+	},
+	"emojis": [
+		{
+			"_id": "CNqjEMFlMYlS66VjDQ3u8z07",
+			"animated": false,
+			"chat_id": "ee754b92-da8a-483a-9601-bf44b9bff531",
+			"name": "ban"
+		}
+	],
+	"isDeleted": false,
+	"p": "Documentation Post <:CNqjEMFlMYlS66VjDQ3u8z07>",
+	"pinned": false,
+	"post_id": "8cacf340-182c-4cbe-98e4-807781ca5e22",
 	"post_origin": "home",
-	"type": 1,
+	"reactions": [
+		{
+			"count": 1,
+			"emoji": "😃",
+			"user_reacted": false
+		}
+	],
+	"reply_to": [
+		{
+			"_id": "d9c07f76-ca61-424c-ae7a-43f1a5f23ade",
+			"attachments": [],
+			"author": {
+				"_id": "Eris",
+				"avatar": "Gi1WvwNobL0X6RpZB7pnAMNw",
+				"avatar_color": "8f75cc",
+				"flags": 4,
+				"pfp_data": 23,
+				"uuid": "d4006f3b-d054-4fd3-a4b1-82b29257cd91"
+			},
+			"emojis": [],
+			"isDeleted": false,
+			"p": "https://tenor.com/view/boom-roasted-the-office-michael-scott-gif-9210948",
+			"pinned": false,
+			"post_id": "d9c07f76-ca61-424c-ae7a-43f1a5f23ade",
+			"post_origin": "home",
+			"reactions": [],
+			"reply_to": [],
+			"stickers": [],
+			"t": {
+				"e": 1723786557
+			},
+			"type": 1,
+			"u": "Eris"
+		}
+	],
+	"stickers": [],
 	"t": {
-		"d": "30",
-		"mo": "01",
-		"y": "2024",
-		"h": "06",
-		"mi": "44",
-		"s": "53",
-		"e": 1706597093
+		"e": 1723786594
 	},
-	"u": "eri",
-	"p": "It’s gonna be great:tm:",
-	"isDeleted": false
+	"type": 1,
+	"u": "ShowierData9978"
 }
 ```
 
-#### Livechat
+#### Livechat (outdated)
```

Comment:
for this and the other instances of things being marked as outdated, could you please actually update the examples